schemas/room-type.json

```diff
@@ -60,8 +60,7 @@
     },
     "description": {
       "description": "Longer room description. Will be displayed to users, and should be in the language indicated by the 'lang' parameter (max length of 1000 characters).",
-      "type": "string",
-      "maxLength": 1000
```

Comment:
Hi @abdollah-aghayan !
I believe the task is to just trim the description, but not remove the validation. I think we can leave this one and just do the trimming in the other PR https://github.com/trivago/ota-entities/pull/25/files